src/util.js

```diff
@@ -495,10 +495,24 @@ Candy.Util = (function(self, $){
 		 *   Emotified text
 		 */
 		emotify: function(text) {
-			var i;
-			for(i = this.emoticons.length-1; i >= 0; i--) {
+			var codeMatches = text.match(/(\`[^`]*\`)/gim) || [];
+			var codeHolders = {
```

Comment:
That was my first solution.
I moved those to be `function`s so that they were more readable since the code is very similar.

However if you think that the functions do not add any readability, we should stick with that simpler approach.